docs_nnx/guides/surgery.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "# Model surgery\n",
     "\n",
-    "In this guide, you will learn how to perform model surgery in Flax NNX using several real-world scenarios:\n",
+    "Model surgery is an act of performing different tasks on the neural network's building blocks and parameters, such as layer replacement, parameter or state manipulation, or even \"monkey patching\". In this guide, you will learn how to perform model surgery in Flax NNX using several real-world scenarios:\n",
```

Comment:
Or maybe simpler ?

```suggestion
    "Model surgery is usually referred to as performing "surgery" on your neural network, such as layer replacement, parameter or state manipulation, or even \"monkey patching\". In this guide, you will learn how to perform model surgery in Flax NNX using several real-world scenarios:\n",
```

@IvyZX @cgarciae @levskaya LMKWYT

Reply:
As discussed with @IvyZX 

"Model surgery is an act of making modifications on an existing neural network's building blocks and parameters, such as layer replacement, parameter or state manipulation, or even "monkey patching"..."